modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -1251,6 +1251,31 @@ CV__DNN_INLINE_NS_BEGIN
     /** @overload */
     CV_EXPORTS_W void blobFromImagesWithParams(InputArrayOfArrays images, OutputArray blob, const Image2BlobParams& param = Image2BlobParams());
 
+    /** @brief Get rectangle coordinates in original image system from rectangle in blob coordinates.
+     *  @param rBlob rect in blob coordinates.
+     *  @param image input inserted in blob.
+     *  @param param struct of Image2BlobParams, contains all parameters needed by processing of image to blob
+     *  @returns rectangle in original image coordinates.
+     */
+    CV_EXPORTS_W Rect blobRectToImage(Rect rBlob, InputArray image, Image2BlobParams param);
```

Comment:
Method signature is different with it's name. I read this as "blob" or "blob's rectangle" as input and "image" as output (because of `ToImage`). But it returns not an image but another Rect.

Reply:
My preference is something like `locateImage` or `locateImageFromBlob` (or `locateBlobFromImage` depends on what is a final Rect`). Similar to [locateROI](https://docs.opencv.org/5.x/d3/d63/classcv_1_1Mat.html#a40b5b3371a9c2a4b2b8ce0c8068d7c96)

Reply:
when blob is build from image for object model inference result (rect) are in blob coordinates.
for example here https://github.com/opencv/opencv_zoo/blob/main/models/object_detection_yolox/demo.py#L56

Method name could be blobRectToImageRect or locateRectFromBlob